userguide/sharing.rst

```diff
@@ -34,13 +34,15 @@ system to meet the requirements of the network.
 
 These types of shares and services are available:
 
-* :ref:`AFP <Apple (AFP) Shares>`: Apple File Protocol shares are
-  often used when the client computers all run macOS. Apple has
-  slowly shifted to preferring :ref:`SMB <Windows (SMB) Shares>` for
-  modern networks, although Time Machine still requires AFP.
+* :ref:`AFP <Apple (AFP) Shares>`: Apple Filing Protocol shares are
+  used when the client computers all run macOS. Apple has deprecated
+  AFP in favour of :ref:`SMB <Windows (SMB) Shares>` and using AFP
+  in modern networks is no longer recommanded (except for providing a
+  Time Machine, for which %brand% currently only supports AFP, not
+  yet SMB).
```

Comment:
This sentence is very confusing and probably needs to be split into at least two sentences.

Reply:
I've reworked this whole section.

Reply:
How does this sound to you:

* :ref:`AFP <Apple (AFP) Shares>`: Apple Filing Protocol shares are accessible from macOS computers (or from other operating systems with third party software). As of Mac OS X 10.9, Apple has deprecated AFP in favor of :ref:`SMB <Windows (SMB) Shares>`, and so use of AFP is no longer recommended unless 1) you need to support pre-10.9 clients or 2) you want to provide a Time Machine (since %brand% does not yet support Time Machine via SMB).

Reply:
Again, this is a really long sentence.  We also try to avoid the informal "you" and asides.  So let me try some rewording:
```:ref:`AFP <Apple (AFP) Shares>`: Apple Filing Protocol shares are accessible from macOS computers or other operating systems with AFP software. Apple deprecated AFP at the release of Mac OS X 10.9 and recommends using :ref:`SMB <Windows (SMB) Shares>` instead. Use of AFP is not recommended unless pre-10.9 Mac clients must be supported or Apple's Time Machine, which requires AFP, is to be used.

(Note: I'm not sure about that last part. I thought Time Machine on SMB worked now.)

Reply:
That wording is mostly fine with me, though your last sentence doesn't make clear that it's FreeNAS that forces AFP use, not Time Machine itself, how about:

"Use of AFP is not recommended unless pre-10.9 Mac clients must be supported or to provide a Time Machine, which %brand% currently only supports via AFP."

As best as I can tell SMB is not supported for Time Machine:
https://redmine.ixsystems.com/issues/23359

If you review the commit message, this lack of support is really the main reason for this patch. :)

I'm on the fence between buying a Synology or FreeNAS, mostly because of this.

Once FreeNAS supports Time Machine via SMB, there's no good reason to enable AFP, as it's only needed for OS X versions that don't even receive security updates from Apple anymore.

Reply:
Time machine over SMB is possible in 11.1. I tested it a few weeks ago. Backup completed successfully. It just requires some auxiliary parameters.

Reply:
Interesting. Where can I read about these 'some auxiliary parameters'?